src/cli/run_command.zig

```diff
@@ -241,6 +243,19 @@ pub const RunCommand = struct {
 
         try replacePackageManagerRun(&copy_script, script);
 
+        if (package_json) |pkg_json| {
+            var i: usize = 0;
+            const prefix = "$npm_package_config_";
+            while (std.mem.indexOfPos(u8, copy_script.items, i, prefix)) |start| {
```

Comment:
this is inlining all the package.json config environment variables from the shell command? it won't be very reliable this way, but i'm not sure why this is necessary